app/src/main/java/com/keylesspalace/tusky/entity/Account.kt

```diff
@@ -45,6 +46,13 @@ data class Account(
             localUsername
         } else displayName
 
+    val fullUsername: String
+        get() {
+            val domain = getDomain(this.url)
+            val localUsername = this.username
+            return "@$localUsername@$domain"
```

Comment:
I'm afraid but this doesn't give the correct result. 
`this.username` is the `acct` of the [Mastodon Account object](https://docs.joinmastodon.org/entities/Account/) "The Webfinger account URI. Equal to username for local users, or username@domain for remote users." 
I think you want the `username` of the Mastodon Account ("The username of the account, not including domain.") which is `this.localusername`.

Second, minor problem that we can probably ignore for now is that there are Mastodon instances where the local domain is not the web domain. Check e.g `@Sir_Boops@sergal.org` where the domain is https://mastodon.sergal.org/@Sir_Boops so parsing the domain part out of the url won't work.

Reply:
What I would do is 
- If it is a local account (same instance as logged in user), construct the full name out of acct + active domain 
- If it is a remote user, acct is the full name, just use that

Reply:
You're right, I mean to do localUsername. The other thing is ugly, glad you caught that D:

Unrelated to this issue: We use `getDomain` in the account-pane "block instance" feature. Is this a problem?

Reply:
Wait, how do I tell if it's the same instance of the logged in user or not? I'm not finding a relevant flag in `Account`.

I could check `username` for an internal `@` and assume if there's no @ they're local, but I think this will misbehave if a server ever misreports the `username` field and leaves the domain out.

Reply:
Yes, check for @. If servers don't comply to the api I'd say it is not our problem.